config/crd/kustomization.yaml

```diff
@@ -13,13 +15,17 @@ patchesStrategicMerge:
 - patches/webhook_in_edgedevices.yaml
 - patches/webhook_in_edgedeployments.yaml
 #- patches/webhook_in_edgedevicegroups.yaml
+#- patches/webhook_in_edgeconfigs.yaml
+#- patches/webhook_in_playbookexecutions.yaml
 #+kubebuilder:scaffold:crdkustomizewebhookpatch
 
 # [CERTMANAGER] To enable webhook, uncomment all the sections with [CERTMANAGER] prefix.
 # patches here are for enabling the CA injection for each CRD
 - patches/cainjection_in_edgedevices.yaml
 - patches/cainjection_in_edgedeployments.yaml
 #- patches/cainjection_in_edgedevicegroups.yaml
+#- patches/cainjection_in_edgeconfigs.yaml
```

Comment:
these are really needed?

Reply:
Honestly.., I don't know. They were generated.
Do you think It's better to remove them and eventually add them in future PR (if needed)?